packages/block-library/src/image/edit.js

```diff
@@ -438,10 +445,11 @@ class ImageEdit extends Component {
 			);
 		}
 
-		const classes = classnames( className, {
```

Comment:
Any reason for the removal of `className`? I think this might be useful when people add a custom className right?